rtsf-at-checkout-loss-detector/main.go

```diff
@@ -7,32 +7,66 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/edgexfoundry/app-functions-sdk-go/appsdk"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	clientInterfaces "github.com/edgexfoundry/go-mod-core-contracts/v2/clients/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
 
 	"loss-detector/functions"
-
 )
 
 const (
 	serviceKey = "LossDetector"
 )
 
+type lossDetectorAppService struct {
+	service            interfaces.ApplicationService
+	lc                 logger.LoggingClient
+	notificationClient clientInterfaces.NotificationClient
+	subscriptionClient  clientInterfaces.SubscriptionClient
+	// serviceConfig *config.ServiceConfig
+}
```

Comment:
I don't see the value of creating this wrapper when every thing in it is available through the a APIs
The template has this wrapper since it has addition custom config which this doesn't need.
Keep it simple until you can't ;-)